addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15977,7 +15977,7 @@ msgstr ""
 #. name of a stereoscopic mode
 #: system/settings/settings.xml
 msgctxt "#36509"
-msgid "Monoscopic (2D)"
+msgid "Monoscopic - 2D"
```

Comment:
if you change this, you also have to change all references to it (like in the settings description), but 2D is not a subtype of moniscopic or vice versa. If you want to get rid of the braces, more correct would be "Monoscopic / 2D" as monoscopic is just the more technical correct term for 2D